cmd/virt-launcher/qemu-kube

```diff
@@ -66,18 +62,27 @@ if [ -z "$CONTAINER_ID" ]; then
 fi
 
 CONTAINER_PID=$(docker inspect --format '{{.State.Pid}}' $CONTAINER_ID)
+echo "container pid: $CONTAINER_PID" >> $LOG
 CMD="$QEMU $ARGS"
 
 date >> $LOG
-echo "$CMD" >> $LOG
+echo "args: $CMD" >> $LOG
+
+# Find out the cgroup path of the container
+CGROUP_PATH=$( cat /proc/${CONTAINER_PID}/cgroup | head -1 | cut -f3 -d: )
+# Get cgroups, don't change systemd and devices cgroup
+CGROUPS=$( cat /proc/${CONTAINER_PID}/cgroup | cut -f2 -d: | grep -v -e systemd -e devices | paste -sd "," - )
```

Comment:
I like this! It just occurred to me that if this script is unable to find the proper container ID (which indicates something's wrong anyways), it won't be able to locate the CONTAINER_PID, both these CGROUP_ lookup lines will also fail, and the script will ultimately fail because ":" isn't a valid cgroup descriptor. I think we should re-visit the decision to not use "set -e" in this script.